### ISSUE TITLE
Left ventricular thrombus

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10103]
**Preferred term label:**
Left ventricular thrombus

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0587044

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Left ventricular thrombus
Myocardial infarction, acute
Cardiomyopathy, takotsubo

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000066
- label: Left ventricular thrombus
- parents: [Abnormal left ventricle morphology (HP:0001711)](https://hpo.jax.org/app/browse/term/HP:0001711) | [Abnormal thrombosis (HP:0001977)](https://hpo.jax.org/app/browse/term/HP:0001977)
- pmid: PMID:33850917
- def: A blood clot (thrombus) localized in the left ventricle. Left ventricular thrombus is a serious complication of acute myocardial infarction and also of non-ischemic cardiomyopathies.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10103
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
correcting HPO id to HP:0040412
